### DISCUSSION TITLE
Is it possible via API to see which authentication providers are active?

### DISCUSSION THREAD

**Discussion by silicia-apps** [appwrite/appwrite#8926] [Q&A]
Hi, 
I'm writing an angular library for developing hybrid applications with appwrite, I was wondering if via api it is possible to retrieve from the server which authentication providers are active in order to automate the rendering of the login form.

**Comment by Saakhavat-Prime** [comment]
At the moment, there isn't an API endpoint that returns the list of authentication providers currently enabled for a project.

If your library needs to render the login form automatically, you'll need to maintain that configuration on the client (or expose it through your own backend) and render the available providers accordingly.

Having an endpoint that exposes the enabled authentication providers would be a useful feature for SDKs and UI libraries, as it would allow applications to dynamically build their login UI without duplicating configuration. If that's your use case, it would make a good feature request.
